src/Microsoft.ML.ImageAnalytics/Imager.cs

```diff
@@ -0,0 +1,420 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using SkiaSharp;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.ML.Data
+{
+    /// <summary>
+    /// Provide interfaces for imaging operations.
+    /// </summary>
+    public class Imager : IDisposable
```

Comment:
The name of this strikes me as unusual.  Why not just call it an "Image".  To me "Imager" implies that its a tool/utility that works with images but not an image itself, however I see that this type wraps a single image.

Reply:
I'll rename it to `Image` then. I was seeing `Image` is generic enough name I was trying to avoid. I think would be ok as long as we are keeping it inside our namespace.

Reply:
Here are some ideas: MLImage, ImageData, ImageBuffer (since the class’s purpose seems to be mostly about exposing the bytes), ImageBytes, ImageMemory, LoadedImage.

Reply:
I'll go with `MLImage`.

Reply:
This is done!